pyproject.toml

```diff
@@ -54,11 +54,16 @@ requires-python = ">=3.11"
 urls = {"Homepage" = "https://github.com/quinnreynolds/minplascalc"}
 
 [tool.hatch.build.targets.sdist]
-include = ["minplascalc/data/*/*.json"]
+include = [
+  "data/*/*.json",
+  "minplascalc/*.py"
+]
 
 [tool.hatch.build.targets.wheel]
 packages = [
-  "minplascalc"
+  "minplascalc",
```

Comment:
These paths are specified from the project root, so thre is no `minplascalc/minplascalc` directory